perun-core/src/main/java/cz/metacentrum/perun/taskslib/dao/jdbc/ServiceDenialDaoJdbc.java

```diff
@@ -24,17 +24,23 @@
 	@Override
 	public void blockServiceOnFacility(int serviceId, int facilityId) throws InternalErrorException {
 		int newBanId = Utils.getNewId(this.getJdbcTemplate(), "service_denials_id_seq");
+		// jdbc template cannot be null
+		//noinspection ConstantConditions
```

Comment:
Wouldn't it be better to use `@SuppressWarnings("ConstantConditions")` instead of this confusing comment? Also, it's really scary that a comment have some functionality.

Reply:
This change was suggested by me. The `//noinspection` is an IntelliJ thing and the reason why I suggested it is because its range is smaller. The suppress on the whole method might hide some other warning in the future. But I agree it might not be the best solution since it only works in IDEA. But, if you think the `@SuppressWarning` is a better solution, it should be changed.